File: examples/java-maven-example/src/test/java/org/galaxio/performance/picatinny/feeders/HttpIntegrationFeeders.java
package org.galaxio.performance.picatinny.feeders;

import org.galaxio.gatling.javaapi.Feeders;

import java.time.format.DateTimeFormatter;
import java.util.Iterator;
import java.util.Map;

public final class HttpIntegrationFeeders {
    private HttpIntegrationFeeders() {}

    public static Iterator<Map<String, Object>> ts() {
        return Feeders.CurrentDateFeeder("ts", DateTimeFormatter.ofPattern("yyyyMMddHHmmssSSS"));
    }
}

File: examples/kotlin-gradle-example/src/gatling/kotlin/org/galaxio/performance/picatinny/feeders/HttpIntegrationFeeders.kt
package org.galaxio.performance.picatinny.feeders

import org.galaxio.gatling.javaapi.Feeders
import java.time.format.DateTimeFormatter

object HttpIntegrationFeeders {
    fun ts() = Feeders.CurrentDateFeeder("ts", DateTimeFormatter.ofPattern("yyyyMMddHHmmssSSS"))
}
